tools/workbench/src/main/java/org/eclipse/rdf4j/workbench/util/WorkbenchRequest.java

```diff
@@ -62,22 +61,36 @@ public class WorkbenchRequest extends HttpServletRequestWrapper {
 	 * @param defaults   application default parameter values
 	 * @throws RepositoryException if there is an issue retrieving the parameter map
 	 * @throws IOException         if there is an issue retrieving the parameter map
-	 * @throws FileUploadException if there is an issue retrieving the parameter map
+	 * @throws ServletException    if there is an issue retrieving the parameter map
 	 */
 	public WorkbenchRequest(Repository repository, HttpServletRequest request, Map<String, String> defaults)
-			throws RepositoryException, IOException, FileUploadException {
+			throws RepositoryException, IOException, ServletException {
 		super(request);
 		this.defaults = defaults;
 		this.decoder = new ValueDecoder(repository,
 				(repository == null) ? SimpleValueFactory.getInstance() : repository.getValueFactory());
 		String url = request.getRequestURL().toString();
-		if (ServletFileUpload.isMultipartContent(this)) {
+		if (isMultipartContent(this)) {
 			parameters = getMultipartParameterMap();
 		} else if (request.getQueryString() == null && url.contains(";")) {
 			parameters = getUrlParameterMap(url);
 		}
 	}
 
+	private static boolean isMultipartContent(HttpServletRequest request) {
+		if (!"POST".equalsIgnoreCase(request.getMethod())) {
+			return false;
+		}
+		String contentType = request.getContentType();
+		if (contentType == null) {
+			return false;
+		}
+		if (contentType.toLowerCase(Locale.ENGLISH).startsWith("multipart/")) {
```

Comment:
I think the multipart/ must always be lowercase when coming from a header so my gut feeling is that enforcing a lowercase is not needed and may even be a point of failure in the future. e.g. badly configured method filtering on a proxy assuming the lowercase rule being by passed by accepting uppercase headers.

Reply:
I just in-lined the code from commons-fileupload when removing the dependency, to ensure we don't break anything. Is that a good enough argument? IMO better safe than sorry, even if I tend to agree with you. ;-)